File: InventoryManagementApp/InventoryManagement.Service/Models/Enrols/DamageModel.cs
using InventoryManagement.Sql.Entities.Enrols;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InventoryManagement.Service.Models.Enrols
{
    public class DamageModel:MasterModel
    {
        public long? ItemId { get; set; }
        public ItemModel Item { get; set; }
        public double Quantity { get; set; }
        public double DamageQuantity { get; set; }
        public string DamageReason { get; set; }
        public long? CompanyId { get; set; }
        public CompanyModel Company { get; set; }
        public DateTime DamageDate { get; set; }
    }
}

File: InventoryManagement-Front/src/app/features/models/damage.model.ts
import { Company } from "./company.model";
import { Item } from "./item.model";

export interface Damage{
    id:number;
    itemId:number;
    quantity:number;
    damageQuantity:number;
    damageReason:string;
    companyId:number;
    damageDate:string;

    item:Item;
    company:Company;
}